api_loader.go

```diff
@@ -360,6 +364,14 @@ func processSpec(spec *APISpec, apisByListen map[string]int,
 		useCoProcessAuth := EnableCoProcess && mwDriver != apidef.OttoDriver && spec.EnableCoProcessAuth
 		useOttoAuth := !useCoProcessAuth && mwDriver == apidef.OttoDriver && spec.EnableCoProcessAuth
 
+		if spec.UseMutualTLSAuth {
```

Comment:
same as above, and you can either remove the log or move it into the mw's `EnabledForSpec`